src/smhi/metobs.py

```diff
@@ -323,6 +323,18 @@ def __init__(
         """
         super().__init__()
 
+        if period == "not-set":
+            ordering = {
+                "latest-hour": 0,
+                "latest-day": 1,
+                "latest-months": 2,
+                "corrected-archive": 3,
+            }
+            available_periods = sorted(
+                periods_in_station.data, key=lambda x: ordering.__getitem__(x)
+            )
```

Comment:
Can we do this into metobs_model possibly using an `IntEnum`?

Something like this, maybe? Maybe we could use pydantic to do the sorting?

```python
from enum import IntEnum

class Period(IntEnum):
    hour = 0
    day = 1
    months = 2
    archive = 3
```

and sort on that?

Reply:
I don't know how to do that.